### PR TITLE
restruct docs and add service deploy charts

### PR DESCRIPTION
**Pull request by qiaoshuangshuang**

### CODE REVIEW

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    
+    job:
+      reclaim:
+        isCleanJob: true
+        isSkipCleanFailedJob: false
+        jobTTLSeconds: 600
+      schedulerName: {{  .Values.paddleflowServer.schedulerName  }}
+      scalarResourceArray:
+        - "nvidia.com/gpu"
+        - "baidu.com/v100_cgpu"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Internal sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+    flavour:
+      - name: flavour1
+        cpu: 1
+        mem: 1Gi
+      - name: flavour2
+        cpu: 1
+        mem: 1G
+        scalarResources:
+          "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Sensitive Information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-jobyaml-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    apiVersion: sparkoperator.k8s.io/v1beta2
+    kind: SparkApplication
+    metadata:
+      name: sparkName
+      namespace: default
+    spec:
+      driver:
+        serviceAccount: spark
+      executor:
+      image: iregistry.baidu-int.com/bmlc/image-packges/spark:pyml-2.8.0
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-jobyaml-configmap.yaml]: Sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      - name: flavour3
+        cpu: 1
+        mem: 8Gi
+        scalarResources:
+          "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Sensitive Information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+      - name: flavour4
+        cpu: 1
+        mem: 8Gi
+        scalarResources:
+          "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Sensitive Information.

File: installer/charts/values.yaml.template [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  printVersionAndExit: ${PADDLEFLOW_SERVER_PRINTVERSIONANDEXIT}
+  image: ${IMAGE_REGISTRY}/${PADDLEFLOW_SERVER_IMAGE}
+  schedulerName: ${VOLCANO_SCHEDULER_NAME}
+  tokenExpirationHour: ${TOKEN_EXPIRATION_HOUR}
+fs:
+  host: paddleflow-fs
+  rpcPort: ${PADDLEFLOW_FS_RPC_PORT}
+  restPort: ${PADDLEFLOW_FS_REST_PORT}
+
+km_volcano:
```
Comment (D0m021ng) [inline installer/charts/values.yaml.template]: remove `km_volcano`

File: installer/charts/volcano/config/volcano-scheduler-pf.conf [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+- plugins:
+  - name: priority
+  - name: gang
+  - name: conformance
+- plugins:
+  - name: drf
+  - name: kmpredicates
+    arguments:
+      kmpredicate.GPUTopoEnable: true
+      kmpredicate.AvailableGPUTypes: "baidu.com/gpu_p40_8,baidu.com/v100_cgpu,baidu.com/p40_cgpu"
```
Comment (D0m021ng) [inline installer/charts/volcano/config/volcano-scheduler-pf.conf]: Sensitive Information.

File: installer/charts/volcano/config/volcano-scheduler-pf.conf [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  - name: proportionpf
+  - name: nodeorder
+  - name: binpack
+    arguments:
+      binpack.weight: 10
+      binpack.cpu: 3
+      binpack.memory: 1
+      binpack.resources: baidu.com/p40_cgpu, baidu.com/p40_cgpu_core, baidu.com/p40_cgpu_memory, baidu.com/v100_cgpu, baidu.com/v100_cgpu_core, baidu.com/v100_cgpu_memory
```
Comment (D0m021ng) [inline installer/charts/volcano/config/volcano-scheduler-pf.conf]: Sensitive Information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 1 above)
Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 2 above)
Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-jobyaml-configmap.yaml (hunk 3 above)
Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-jobyaml-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 4 above)
Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 5 above)
Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

Comment (qiaoshuangshuang) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/values.yaml.template (hunk 6 above)
Comment (qiaoshuangshuang) [reply installer/charts/values.yaml.template]: done

**Review by D0m021ng — APPROVED** [review]

File: installer/charts/volcano/config/volcano-scheduler-pf.conf (hunk 7 above)
Comment (qiaoshuangshuang) [reply installer/charts/volcano/config/volcano-scheduler-pf.conf]: done

File: installer/charts/volcano/config/volcano-scheduler-pf.conf (hunk 8 above)
Comment (qiaoshuangshuang) [reply installer/charts/volcano/config/volcano-scheduler-pf.conf]: done

**Pull request merged**